content/ja/tracing/trace_collection/dd_libraries/dotnet-core.md

```diff
@@ -118,7 +125,7 @@ AWS Lambda で Datadog APM を設定するには、[サーバーレス関数の
 Azure App Service で Datadog APM を設定するには、[Azure App Service 拡張のトレース][1]を参照してください。
 
 [1]: /ja/serverless/azure_app_services/
-{{< /tabs >}}
+{{% /tab %}}
```

Comment:
@davidejones just for awareness, are the delimiters for the shortcodes diff because they're nested inside another tab?

Reply:
Yea so the tabs and tab shortcode are intentionally different kinds and intentionally nested. 

The problem was this newer hugo version has some better checking of unclosed shortcodes and It picked up on this translated page as having issues. 

It seems like it was using the plural `tabs` instead of `tab` to close a single tab and then it was opening with `%` but then closing with `<` so i corrected these.

It was likely caused by translators making a mistake on this page and no one noticed and hugo didn't error like it should at the time.